pyvrp/solve.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pyvrp.GeneticAlgorithm import GeneticAlgorithm
+from pyvrp.PenaltyManager import PenaltyManager
+from pyvrp.Population import Population
+from pyvrp._pyvrp import ProblemData, RandomNumberGenerator, Solution
+from pyvrp.crossover import ordered_crossover as ox
+from pyvrp.crossover import selective_route_exchange as srex
+from pyvrp.diversity import broken_pairs_distance as bpd
+from pyvrp.search import LocalSearch, compute_neighbours
+
+if TYPE_CHECKING:
+    import pathlib
+
+    from pyvrp.Result import Result
+    from pyvrp.stop import StoppingCriterion
```

Comment:
This can go all the way down to just above `class SolveParams:`, right?

Reply:
(The imports below and above seem to be a bit out of whack due to this conditional in the middle)